ci-operator/step-registry/cucushift/upgrade/eus/pause-worker-mcp/cucushift-upgrade-eus-pause-worker-mcp-ref.yaml

```diff
@@ -8,6 +8,10 @@ ref:
   grace_period: 10m
   commands: cucushift-upgrade-eus-pause-worker-mcp-commands.sh
   cli: latest
+  env:
+  - name: PAUSED_MCP_NAME
+    default: "worker"
```

Comment:
does ${PAUSED_MCP_NAME} only support pausing one pool?

Reply:
Ah, similar comment with https://github.com/openshift/release/pull/55328#discussion_r1723016450

Reply:
I am thinking are there any difference between pause one mcp and multiple mcp? if the only difference is pause multiple mcp have to unpause them one by one, then I think it really don't need to test multiple mcp.

Reply:
Per my understanding, canary upgrade is designed for customers to create several custom mcp for upgrading worker nodes batch by batch, it's also consistent with the official [doc](https://docs.openshift.com/container-platform/4.16/updating/updating_a_cluster/update-using-custom-machine-config-pools.html#example_update-using-custom-machine-config-pools).

Reply:
Implemented it in the new commit